### ISSUE TITLE
🐛 [Bug]: UI components are missing accessibility labels

### ISSUE THREAD

**Issue by mhussaincov94** [serge-chat/serge#1057]
hi, i'm blind and use orca the screen reader on linux.
I have some accessibility issues.
if it is possible could you look in to these?
I would be very greatful.

I will list them below
the combo box where you can choose the moddle to use, is not accessible to orca the screen reader on linux.
I know that there is a dropdown box and i can enter it, I can not select a moddle, due to the moddle names not being red out.
2.
there are 2 checkboxes.
1 is unlabled and i'm not sure what the first one does.
the moddle settings checkbox is identifyed correctly.
when the settings link is pressed.
3.
the button to regenerate the responce from the moddle is not labled. could this be labled?
orca the screen reader only says button.
there is also a final button that is not labled at all, it comes after the edit box where you type things.
and is just before this.
Serge - Powered by LLaMA
if it helps the button which starts a new chat is accessible with orca.
non accessibility.
last issue.
changeing moddle settings could a button which is labled be added which will save settings that have been changed by the user?
currently I don't believe the settings get changed.
please correct me if i'm incorrect.
thank you for serge!

running debian testing with all updates installed.
running the latest version of chromium from the debian repos.
running docker via there repo.
running serge via docker.

**Comment by morpheus2448** [comment]
@mhussaincov94
Thanks for the issue, I'll look at this after my next PR!

**Comment by mhussaincov94** [comment]
my thanks! off topic, but your name is simply ausom. perchance have you been in charge of my dreams sir morpheus?
beware the endless...
sorry bout that.
thank you for looking in to this. and forgive my levity.

**Comment by gaby** [comment]
Fixed #1136
